test/parallel/test-http-set-timeout-server.js

```diff
@@ -213,3 +213,17 @@ test(function fastTimeout(cb) {
     });
   }));
 });
+
+test(function serverTimeout(cb) {
+  const server = http.createServer();
+  server.listen(common.mustCall(() => {
+    const timeoutValue = 50;
+    const s = server.setTimeout(timeoutValue);
+    assert.strictEqual(s.timeout, timeoutValue);
+    assert.ok(s instanceof http.Server);
+    http.get({
+      port: server.address().port
+    }).on('error', common.mustCall());
```

Comment:
@trivikr - IIUC, there are two reasons why the request would fail:
  - `ECONNRESET` - as the server does not have a client handler
 - `ECONNREFUSED` - as the server is unavailable, due to the `close` in the next line.

right now looks like the second one is masking the first one? what is the original intent? I am unable to open the referenced link in the PR, and hence asking.

Reply:
This code was written during SeattleJS December 2018 meetup.
I just revisited the video in which I identify the branch which this test covers https://www.youtube.com/watch?v=GuVEuz_pD5k&t=16m15s

Looks like this branch is already covered in latest coverage, so this test is no longer required https://coverage.nodejs.org/coverage-779d7efa72ed514a/lib/_http_server.js.html#L347